### DISCUSSION TITLE
What is this doing here?

### DISCUSSION THREAD

**Discussion by dxlbnl** [Dan6erbond/sk-auth#43] [General]
https://github.com/Dan6erbond/sk-auth/blob/731eabd893f29df64870beb7ce65282312852da0/src/auth.ts#L193

**Comment by Dan6erbond** [comment]
Hey there! Thanks for posting. The intention is at some point to add a CSRF route that generates a CSRF token for a one-time use so that direct logins via HTTP POST can be checked with a CSRF token, if that use-case is required. At the moment I just left that there as a to-do. NextAuth.js implements similar functionality so I wanted to bring it over to SKAuth.
